enterprise/api_client/enterprise.py

```diff
@@ -30,19 +30,20 @@ class EnterpriseApiClient(JwtLmsApiClient):
 
     DEFAULT_VALUE_SAFEGUARD = object()
 
-    def get_content_metadata(self, enterprise_customer):
+    def get_content_metadata(self, enterprise_customer, catalogs_to_transmit=None):
```

Comment:
nit: if we're adding this argument here, it would be better to name it simply `enterprise_catalogs` because this code is not specific to transmitting catalog data to a third party.